Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -298,7 +307,38 @@ -(void)filterContentForSearchText:(NSString*)searchText scope:(NSString*)scope {
     NSPredicate *predicate = [NSPredicate predicateWithFormat:@"(SELF.title LIKE[cd] %@)", searchWithWildcards];
     
     self.filteredEventsArray = [NSMutableArray arrayWithArray:[self.allEvents filteredArrayUsingPredicate:predicate]];
```

Comment:
also, at line 290
```objective-c
// Remove all objects from the filtered search array
[self.filteredEventsArray removeAllObjects];
```
Is this necessary? We are reassigning it here. Maybe there is some performance thing, but I'm not sure how ARC would handle this. You should investigate.